Source/DataSources/ModelVisualizer.js

```diff
@@ -327,6 +327,12 @@ define([
     function onModelError(error) {
         console.error(error);
     }
+    function checkModelLoad(model, entity, modelHash){
+        model.readyPromise.otherwise(function(error){
+            onModelError(error);
```

Comment:
It's fine now to just copy the contents of `onModelError` into here and remove that function.